### PR TITLE
Replaces RunConfiguration and Configuration with launchSettings.json

### PR DESCRIPTION
**Pull request by teinarss**

### CODE REVIEW

File: OpenRA.Game/OpenRA.Game.csproj [hunk 1]
```diff
@@ -73,6 +49,7 @@
     <None Include="App.config" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" /> 
     <AdditionalFiles Include="../stylecop.json" />
+    <AdditionalFiles Include="Properties/launchsettings.json" />
```
Comment (pchote) [inline OpenRA.Game/OpenRA.Game.csproj]: `launchSettings.json`

File: OpenRA.Game/OpenRA.Game.csproj [hunk 2]
```diff
@@ -27,30 +27,6 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <Optimize>false</Optimize>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Red Alert' OR '$(RunConfiguration)' == 'Red Alert' ">
```
Comment (pchote) [inline OpenRA.Game/OpenRA.Game.csproj]: It seems that MonoDevelop/vs:mac don't support `launchSettings.json` after all. Can we please keep the `$(RunConfiguration)`s for that?

**Review by pchote — COMMENTED** [review]
This works nicely in Rider, and will fix the confusion in MD thanks to that not exposing the launchSettings at all with vs/rider not exposing the RunConfigurations at all.

So LGTM with the fixes below. I'll have a followup after this to handle the other MD regressions.

**Comment by teinarss** [comment]
Updated!

**Review by pchote — APPROVED** [review]
Works nicely on Rider, VS:Mac, and MonoDevelop 7.8.2.

Note that VS:Mac and MD require an additional fix for #16534, which I will be PRing alongside some other changes after this.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/83640a19aeaf4362824cc15c37bb1740b2189972...bdbc50df97ab77cb69fa5dbd13427289a423a137)

**Pull request merged**